### PR TITLE
Support blank target in -item message

### PR DESCRIPTION
**Pull request by MathyFurret**
For Gen 4-5 Frisk. If no target is sent with an -item message, the client first checks if there is exactly 1 possible target, and updates its item.

### CODE REVIEW

**Comment by MathyFurret** [comment]
Guess we're on an older TS version, hold on

**Comment by MathyFurret** [comment]
https://youtu.be/kSglNQBFGQg

File: play.pokemonshowdown.com/src/battle.ts [hunk 1]
```diff
@@ -2229,6 +2229,20 @@ export class Battle {
 			let item = Dex.items.get(args[2]);
 			let effect = Dex.getEffect(kwArgs.from);
 			let ofpoke = this.getPokemon(kwArgs.of);
+			if (!poke) {
+				if (effect.id === 'frisk') {
+					const possibleTargets = ofpoke!.side.foe.active.filter(p => p !== null);
```
Comment (urkerab) [inline play.pokemonshowdown.com/src/battle.ts:2234]: Wouldn't it be easier to use `find` instead of `filter`?

Comment (MathyFurret) [reply play.pokemonshowdown.com/src/battle.ts:2234]: find doesn't account for having >1 possible target.

**Pull request merged**